client/package.json

```diff
@@ -10,6 +10,7 @@
     "d3-ease": "^2.0.0",
     "google-map-react": "^2.1.8",
     "moment": "^2.29.1",
+    "mongodb": "^3.6.3",
```

Comment:
We don't need mongodb as a dependency on the frontend, so you can remove this import